packages/core/src/domain/console/consoleObservable.ts

```diff
@@ -6,9 +6,17 @@ import { ConsoleApiName } from '../../tools/display'
 import { callMonitored } from '../../tools/monitor'
 
 export interface ConsoleLog {
+  /* Error message created from the concatenation of console parameters */
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏Nice initiative

<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏to be even more helpful, just use `/** ... */` instead of `/* ... */` so TS picks it up and displays it in your IDE of choice :)

Reply:
👌